### ISSUE TITLE
Senua’s Saga: Hellblade II is the next game that runs better without Nukem9's mod

### ISSUE THREAD

**Issue by moesius** [Nukem9/dlssg-to-fsr3#423]
On my RTX 3060 6GB I get 80 fps without using the mod.
With the mod I get 120 fps but it looks like 40 fps.
Same happens with Horizon Forbidden West.
With any other games the mod works perfectly.

**Comment by Erebus1245** [comment]
Yeah framegen doesn't feel right and what's weird is that the game is even supposed to support FSR3 by default but there is no framegen support onlly DLSSG get's it.

**Comment by NoUserNameForYou** [comment]
How do you install it?

**Comment by NoUserNameForYou** [comment]
How do you install the DLSS tweaks edition into this? The Nvidia plugin folders confuse me. I put the dlls in their respective folders as well as in the binaries folder but I can't. The first game I'm having problems with.

XGP version.

**Comment by Erebus1245** [comment]
Universal version with the dbghelp.dll in Content\Hellblade2\Binaries\WinGDK folder where the main exe is, this is for the game pass version steam version has a different folder names.

**Comment by NoUserNameForYou** [comment]
> Universal version with the dbghelp.dll in Content\Hellblade2\Binaries\WinGDK folder where the main exe is, this is for the game pass version steam version has a different folder name.

Thanks. Got the DLSStweaks version working and having 0 problems apart from the universal no vsync tearing on my 60Hz setup.
My base frame rate was around 70 at 1080p DLSS Q, so 120ing it is easier and the game feels smooth.

**Comment by NoUserNameForYou** [comment]
> Same happens with Horizon Forbidden West.

I have it on jailbroken PS4 Pro with resolution and frame rate patch. Even when the game runs around close 60 FPS, it still feels like 20 ish to 30ish. That engine is borked. I don't think it's an FSR mod issue.

**Comment by Erebus1245** [comment]
Well i tried it in the steam version now instead of the Game pass version i used bevore and it definately feels much better in it dunno maybe something is wrong with game pass version would not be the first time either.

**Comment by TimmyP777** [comment]
There are 2 exe. You want to put the dlls by the one further into directory structure.

**Comment by Shasoosh** [comment]
Worked very purly with the Universal version.

Tried loading the DLSSTweaks version (as someone mentioned here that it works better) but it it wouldn't run. I can see in the DLSSTweaks log that dlssg_dlsstweaks_wrapper.dll is loaded but nothing happens after that.

**Comment by TimmyP777** [comment]
Very poorly? What hardware?

3070 5800x3D 32gb

Straight 50fps gain (60 to 110).

**Comment by 18123052778** [comment]
You need to turn off DLSS sharpening, as it will negate the effect of your frame interpolation.

**Comment by 18123052778** [comment]
> 非常糟糕，就像问题的标题一样，在没有 mod 的情况下运行得更好。没有 mod 为 60 fps，使用 mod 为 -110 fps，但感觉像 30 fps。
> 
> 3080在这里。

You need to turn off DLSS sharpening, as it will negate the effect of your frame interpolation.

**Comment by 18123052778** [comment]
> 在我的 RTX 3060 6GB 上，我在不使用模组的情况下获得了 80 fps。使用模组，我得到 120 fps，但看起来像 40 fps。《地平线：西之绝境》也是如此。对于任何其他游戏，该模组都可以完美运行。

You need to turn off DLSS sharpening, as it will negate the effect of your frame interpolation.

**Comment by Erebus1245** [comment]
Works perfectly for me now in the steam version after some tinkering, some tips:

Disabling DLSS sharpening and motion blur ingame is necessary as it will cause ghosting external post processing sharpening like reshade or from NVCP doesn't cause this.
Same for some of the blur effects and CA in the game this can be fixed by disabling these effects in the engine.ini:


```
[SystemSettings]
r.SceneColorFringeQuality=0
r.Tonemapper.GrainQuantization=0
r.Tonemapper.Quality=0
r.NT.Lens.Distortion.Intensity=0
r.NT.Lens.Distortion.Stretch=0
r.NT.Lens.ChromaticAberration.Intensity=0
r.NT.DOF.RotationalBokeh=0
r.NT.DOF.NTBokehTransform=0
r.FilmGrain=0
r.NT.AllowAspectRatioHorizontalExtension=0
r.NT.EnableConstrainAspectRatio=0


```
If you want to cap the fps change FrameRateLimit in the GameUserSettings.ini i don't recommend using external frame limiters as they can cause some judder.

**Comment by Nick9604** [comment]
Erebus 1245 I'm sorry, which one do you recommend me to use as a universal mod?

**Comment by Erebus1245** [comment]
version.dll

**Comment by Nick9604** [comment]
version.dll i.e. which one? I'm sorry I did not understand

**Comment by Nick9604** [comment]
Do you recommend me to make that change on the engine?

**Comment by Nick9604** [comment]
I still can't find the location of the engine?

**Comment by Erebus1245** [comment]
engine.ini is in AppData\Local\Hellblade2\Saved\Config\Windows for steam  AppData\Local\Hellblade2\Saved\Config\WinGDK for game pass

**Comment by Shasoosh** [comment]
> > 非常糟糕，就像问题的标题一样，在没有 mod 的情况下运行得更好。没有 mod 为 60 fps，使用 mod 为 -110 fps，但感觉像 30 fps。
> > 3080在这里。
> 
> You need to turn off DLSS sharpening, as it will negate the effect of your frame interpolation.

Thanks! That relly helped the feel of the game which is way smoother but now I have really bad ghosting. How can I remove it?

**Comment by rudypangestu** [comment]
Just to let everyone knows that it works perfectly for me, after some tweaks. I use the mod from 'dll_version' folder.

What you need to do:
- Disable motion blur
- Disable sharpening
- Remove letter boxing, chromatic aberration, and film grain by applying these settings:

```
[SystemSettings]
r.SceneColorFringeQuality=0
r.Tonemapper.GrainQuantization=0
r.Tonemapper.Quality=0
r.NT.Lens.Distortion.Intensity=0
r.NT.Lens.Distortion.Stretch=0
r.NT.Lens.ChromaticAberration.Intensity=0
r.NT.DOF.RotationalBokeh=0
r.NT.DOF.NTBokehTransform=0
r.FilmGrain=0
r.NT.AllowAspectRatioHorizontalExtension=0
r.NT.EnableConstrainAspectRatio=0
```

After that, it works perfectly as intended. Not just numbers going up. I've compared it with sharpening turned on, which negates the frame gen but the FPS counter is still high.

**Comment by Shasoosh** [comment]
Captured a video of what I'm getting with the above settings with any fast mouse movement. 

https://youtu.be/9aVMEPxp_3g

**Comment by Erebus1245** [comment]
This just happens if you move your camera really fast with the mouse it's a downside of framegen and normal  atm so just don't do that or play with a gamepad who knows maybe FSR3.1 will improve it a bit when comes out later this year.

**Comment by Shasoosh** [comment]
Thanks. I wasn't sure if others experience the same issue or if it's a misconfiguration on my end. Good point about the controller - I wasn't able to trigger it with that either.

**Comment by LastStageZ** [comment]
> Captured a video of what I'm getting with the above settings with any fast mouse movement.
> 
> https://youtu.be/9aVMEPxp_3g

hey I've had the same issue and it seems that there's no fix so far 
but there is an alternative option tho, you can use lossless scalling 2.8.1 or 2.8.2 they added frame generation 2.0 and it's actually
very good it doesn't cause shimmering or ghosting and it generates a good amount of frames and feels smooth even if your base frame is low (30-40) fps

**Comment by TimmyP777** [comment]
lol

**Comment by D71Gaming** [comment]
The new update killed the mod. Was working fine yesterday. Anyone has any suggestions?
